src/map/utils/battleutils.cpp

```diff
@@ -5023,6 +5032,37 @@ namespace battleutils
         PChar->PClaimedMob = nullptr;
     }
 
+    // Checks to see if the mob has a damage cap value
+    // This is used for instances like Suttung, Antaeus, Crustacean Conundrum bcnm, Colonization reives
+    int32 DamageCapDistribution(int32 damage, CBattleEntity* PDefender)
+    {
+        int32 damageCap     = PDefender->getMod(Mod::RECEIVED_DAMAGE_CAP);     // The max damage cap
+        int32 damageVariant = PDefender->getMod(Mod::RECEIVED_DAMAGE_VARIANT); // The value you want the damage to have a variance by
+
+        // If the target has no mod or the damage is less than the cap return normal damage
+        if ((damageCap == 0) || damage < damageCap)
+        {
+            return damage;
+        }
+        else
+        {
+            // Clamp the damage since the damage is higher than the cap
+            damage = std::clamp(damage, 0, damageCap);
+        }
+
+        // If for whatever reason your damage variant is set too high set the variant to 0 as a fail safe
+        if (damageVariant > damageCap)
```

Comment:
Do you think this would need some warning logging, as this would be user setup error?

Reply:
I added a warning to let them know the variant is too high